karma.conf.js

```diff
@@ -20,9 +19,9 @@ module.exports = function (config) {
             'app/*.module.js',
             'app/*.controller.js',
             'app/*.config.js',
-            'app/*.utils.js',
             'app/kudos-feed/kudos-feed.controller.js',
             'app/components/**/*.component.js',
+            'app/components/**/*.controller.js',
```

Comment:
If you chose to have tests in a standalone folder - might as well make the runner just take ALL the *.js files available.